pkg/controller/acicontainersoperator.go

```diff
@@ -109,6 +112,9 @@ const aciContainersController = "aci-containers-controller"
 const aciContainersHostDaemonset = "aci-containers-host"
 const aciContainersOvsDaemonset = "aci-containers-openvswitch"
 
+var Aci_operator_config_path = "/usr/local/etc/aci-containers/aci-operator.conf"
```

Comment:
while this addresses the earlier comment about predefining the strings, I am still not clear why this path has to be an absolute path whereas earlier it was not...

Reply:
Earlier also it was. The function ReadConfigMap is called like below 

**rawSpec, err := c.ReadConfigMap("aci-containers/aci-operator.conf")**

and inside the function, it is coded with **/usr/local/etc**

**func (c *Controller) ReadConfigMap(field string) ([]byte, error) {
	raw, err := ioutil.ReadFile("/usr/local/etc/" + field)**

So eventually it becomes absolute and we are not changing this location as we mention this location in volume mount in deployment file